8Knot/pages/contributors/visualizations/contribs_by_action.py

```diff
@@ -154,16 +166,31 @@ def toggle_popover(n, is_open):
     background=True,
 )
 def contribs_by_action_graph(repolist, interval, action):
-
     # wait for data to asynchronously download and become available.
-    cache = cm()
-    df = cache.grabm(func=ctq, repos=repolist)
-    while df is None:
-        time.sleep(1.0)
-        df = cache.grabm(func=ctq, repos=repolist)
+    while not_cached := cf.get_uncached(func_name=ctq.__name__, repolist=repolist):
+        logging.warning(f"{VIZ_ID}- WAITING ON DATA TO BECOME AVAILABLE")
+        time.sleep(0.5)
 
+    logging.warning(f"{VIZ_ID} - START")
     start = time.perf_counter()
-    logging.warning(f"{VIZ_ID}- START")
+
+    # GET ALL DATA FROM POSTGRES CACHE
+    df = cf.retrieve_from_cache(
+        VIZ_ID,
+        tablename="contributors",
+        repolist=repolist,
+        column_override=[
```

Comment:
What is column_override? How and why should it be used?

Reply:
okay removed column_override. if we want it later we can make it an optional argument